app/events/email-incoming/index.ts

```diff
@@ -36,6 +36,22 @@ interface UserData {
   submit?: boolean
 }
 
+interface EmailProps {
+  subjectMessage: string
+  userEmail: string
+  body: string
+  parsedSubmissionSubject: string
+  newCircularId?: number
+}
+
+interface EmailProps {
+  subjectMessage: string
+  userEmail: string
+  body: string
+  parsedSubmissionSubject: string
+  newCircularId?: number
+}
```

Comment:
Why are there two copies of this declaration?

Reply:
Looks like it happened in the merge resolver, fixing now